client/config/environment.js

```diff
@@ -37,17 +38,15 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' www.google.com www.google-analytics.com",
```

Comment:
ok I take that back.  We do need it.  I was forgotten that I had basically disabled the real ga code when not in production, when I re-enabled it locally I did again get the policy errors and the pageview code did not fire.  I have added it back in again and verified the ga logic is working in the realtime analytics dashboard.

![image](https://cloud.githubusercontent.com/assets/7982720/8867459/36aab3a0-318f-11e5-8fb8-93d0d990dfaa.png)